Uno.Gallery/Uno.Gallery.macOS/Info.plist

```diff
@@ -26,11 +26,7 @@
 	<string>NSApplication</string>
 	<key>XSAppIconAssets</key>
 	<string>Assets.xcassets/AppIcon.appiconset</string>
-	<key>UIAppFonts</key>
-	<array>
-		<string>Fonts/uno-fluentui-assets.ttf</string>
-		<string>Fonts/roboto.ttf</string>
-		<string>Fonts/SF-Pro.ttf</string>
-	</array>
+	<key>ATSApplicationFontsPath</key>
```

Comment:
@kazo0 Maybe this is something that should be explained in the uno documentation?

Reply:
It's done properly in the application templates, but yeah I will check the docs to see if we mention this